aten/src/ATen/native/quantized/cpu/qtanh.cpp

```diff
@@ -0,0 +1,88 @@
+#include <ATen/ATen.h>
+#include <ATen/NativeFunctions.h>
+#include <ATen/core/op_registration/op_registration.h>
+#include <ATen/native/TensorIterator.h>
+#include <ATen/native/cpu/Loops.h>
+#include <ATen/quantized/Quantizer.h>
+#include <ATen/native/quantized/cpu/quantized_ops.h>
+#include <ATen/native/quantized/cpu/init_qnnpack.h>
+#include <ATen/native/quantized/cpu/qnnpack_utils.h>
+#include <caffe2/utils/threadpool/ThreadPoolMobile.h>
+
+#include <algorithm>
+
+namespace at {
+namespace native {
+
+DEFINE_DISPATCH(qtanh_stub);
+
+#ifdef USE_PYTORCH_QNNPACK
+// This ALWAYS outputs scale=2.0/256, zp=128, dtype=quint8
```

Comment:
Did this change from 1/256.0 and zp=0 because of the symmetric range between [-1, 1]? Curious as to how the tests were passing earlier then

Reply:
There were no tests for the QNNpack -- I enabled them after your previous comment. After implementing the tests, this was caught